keywording/text.xml

```diff
@@ -348,7 +348,7 @@ arches that you would like to stabilize.
 
 <p>
 If your package is architecture-independent, you should add the
-<c>&lt;stabilize-allarches&gt;</c> tag to metadata.xml. This allows consistency
+<c>&lt;stabilize-allarches&gt;/</c> tag to metadata.xml. This allows consistency
```

Comment:
That's the only place where I'd add it.
However:
```suggestion
<c>&lt;stabilize-allarches/&gt;</c> tag to metadata.xml. This allows consistency
```

Reply:
Heh, I guess I was blinded by the `</c>`. Re-did this based on your suggestion.